control/reset.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, CESAR. All rights reserved.
+#
+# SPDX-License-Identifier: BSD 3-Clause
+
+import logging
+import os
+import json
+
+from pkg_resources import resource_string
+from mongo import MongoConn
+from rabbitmq import RabbitMQConn
+
+RESET_SETTINGS_FILE = 'resetSettings.json'
+
+def createReset():
```

Comment:
the file path shouldn't be passed as argument?

Reply:
@jcorde suggested to use it on a defined constant.

Reply:
Why, @ramonhpr ?

Reply:
The createReset is acting like an abstract factory (a design pattern to create an instance of a class by its parameter) and depending on the settings we can have another instance. the default one is to connect with a the local database and rabbitmq of the gateway.
We can have always the same path though but most of daemons let the user pass the configuration file or same flag to change the configuration.

Reply:
_connect with a **the** local_

Reply:
I agree, @ramonhpr . It makes total sense to add it as an argument in cases where the value can be changed.
But I see no way or even reason for us to allow it to be modified by now.

As well mentioned by @joaoaneto : both approaches are correct, being it just a matter of preferences.

Even though, I respect your position and believe that if you say this is the best option, it should be.

Reply:
If @joaoaneto says that both are correct so I agree with him. And we can let it without modification for now.